magento/magento1ce/2018-11-28.yaml

```diff
@@ -0,0 +1,9 @@
+title:     SUPEE-10975 - Security enhancements that help close RCE,XSS,CSRF and other vulnerabilities
+link:      https://magento.com/security/patches/supee-10975
+cve:       ~
+branches:
+    1.x:
+        time:     2018-11-28 00:00:00
```

Comment:
it should be the exact time of the merge commit that fixed the issue.

Reply:
I updated the times.